manifests/init.pp

```diff
@@ -32,6 +32,7 @@
   Optional[String[1]]                       $seven_zip_source   = undef,
   Boolean                                   $aws_cli_install    = false,
   Boolean                                   $gsutil_install     = false,
+  Hash                                      $archives           = {},
```

Comment:
Could you document the parameter using the puppet-strings style as above https://github.com/voxpupuli/puppet-archive/pull/423/files#diff-c6c3a824d185ff01e063011517a1b13a604dc3d7b63d1c67a30451c10df02bcbL26?

I dunno, something like...

```puppet
# @param archives
#   A hash of archive resources that this module should create
```

Bonus points for adding a `@example` to the examples section.

Reply:
Done